qemu/tests/cfg/blockdev_commit_specify_node.cfg

```diff
@@ -0,0 +1,40 @@
+- blockdev_commit_specify_node:
+    type = blockdev_commit_specify_node
+    virt_test_type = qemu
+    only Linux
+    images += " data"
+    force_create_image_data = yes
+    remove_image_data = yes
+    start_vm = yes
+    kill_vm = yes
+    storage_pools = default
+    storage_type_default = "directory"
+    storage_pool = default
+    image_size_data = 500M
+    image_name_data = data
+    snapshot_tags_data = sn1 sn2 sn3 sn4
+
+    image_size_sn1 = 500M
+    image_name_sn1 = sn1
+    image_format_sn1 = qcow2
+
+    image_size_sn2 = 500M
+    image_name_sn2 = sn2
+    image_format_sn2 = qcow2
+
+    image_size_sn3 = 500M
+    image_name_sn3 = sn3
+    image_format_sn3 = qcow2
+
+    image_size_sn4 = 500M
+    image_name_sn4 = sn4
+    image_format_sn4 = qcow2
+
+    device_tag = "data"
+    rebase_mode = unsafe
+    qemu_force_use_drive_expression = no
+    variants:
+        - base_same_with_top:
+            bst_qmp_error_msg = "'desc': 'cannot commit an image into itself'"
+        - parent_as_top_child_as_base:
+            ptcb_qmp_error_msg = "'drive_sn3' is not in this backing file chain"
```

Comment:
bst - binary search tree :) joking
but bst and ptcb are meaningless to your tests, it's OK to cover several cases in one script, but it's better to explicitly specify them, so what about:

- base_same_with_top:
test_scenario = commit_to_the_same_node
error_msg = xxx
- parent_as_top_child_as_base:
test_scenario = commit_parent_node_to_child
error_msg = xxx

define commit_parent_node_to_child and commit_to_the_same_node as two member functions, 
use getattr(self, params['test_scenario']) to get the function object, then call it.
That could make your code clear.

Or it's OK for me to handle it in your way, but I suggest setting a meaningful name